_candidates_en/KenjiDoi_1.html

```diff
@@ -0,0 +1,23 @@
+---
+name: Doi Kenji
+title: "In conclusion how Scala in the project.-Scala development by the beginner group-"
+length: 15
+audience: Beginner
+language: Japanese
+twitter: doiken208
+github: doiken
+icon: https://avatars0.githubusercontent.com/u/6147573?v=3&s=400
+organization: CyberAgent, Inc.
+---
+<p>
+At movie ad network named Lodeo, 6 Scala beginners and 4 members experienced under half a year have developed.
+In this session, I will talk how to develop by Scala in the situation, and the good practice and bad practice while comparing the project start timing between the project mid-term when 2 beginners joined.
+</p>
+<p>
+Themes are following.
```

Comment:
The themes are **the** following: (*"is the following:"* is the idiomatic phrase)